### PR TITLE
Feat: Added wishlisht and cart page

### PR DESCRIPTION
**Pull request by shahbazibnafzal**
1. Added wishlist and cart page
2. Added link for wishlist and cart page in navbar of all the pages

### CODE REVIEW

**Comment by Hamza-777** [comment]
The logo on wishlist and cart page is not redirecting to the main page, not that it is very important now, or maybe you meant it to be that way.

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
+@import url(master.css);
+
+.cart-grid-container {
+    margin:0;
+    padding: 0;
```
Comment (Hamza-777) [inline css/cart.css:5]: Can be handled with global css.

**Review by Hamza-777 — COMMENTED** [review]
Looks good!

**Review by vrx29 — APPROVED** [review]
Nice Ui. Looks good to me.

**Review by Hamza-777 — APPROVED** [review]

**Review by BalajiHarinath — APPROVED** [review]
Awesome UI. Looks good to me.

**Comment by shahbazibnafzal** [comment]
> The logo on wishlist and cart page is not redirecting to the main page, not that it is very important now, or maybe you meant it to be that way.

Added the link to home in wishlist and cart page. Thanks for pointing this out.

**Comment by shahbazibnafzal** [comment]
Thanks @vrx29 , @Hamza-777 and @BalajiHarinath  for the approval. Merging it to dev.

**Pull request merged**